app/Jobs/ApplicationDeploymentJob.php

```
@@ -1402,7 +1402,7 @@ private function generate_buildtime_environment_variables()
         }
 
         $envs = collect([]);
-        $coolify_envs = $this->generate_coolify_env_variables();
+        $coolify_envs = $this->generate_coolify_env_variables(forBuildTime: true);
```

PHPMD (2.15.0)

The variable $coolify_envs is not named in camelCase. (undefined)

(CamelCaseVariableName)